service/templates/admin.html

```diff
@@ -1297,20 +1297,24 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 										<input type="text" ng-model="Settings.domain_name">
 									</md-input-container>
 									<md-input-container class="control" style="width:45%;">
-										<label>Github PR creator organization</label>
-										<input type="text" ng-model="Settings.github_pr_creator_org">
+										<label>Github Organizations</label>
+										<textarea ng-model="Settings.github_orgs" ng-list="&#10;" ng-trim="false"  md-select-on-focus></textarea>
```

Comment:
Sorry if I'm missing this, but how does this because a string slice?

Reply:
Also something I did not know, but learned from superusers: `ng-list="&#10;"` splits the input on newline.

Reply:
Whoa cool.